FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -226,6 +228,15 @@ - (NSString *)_fileNameForSelector:(SEL)selector
   if (0 < identifier.length) {
     fileName = [fileName stringByAppendingFormat:@"_%@", identifier];
   }
+  
+  if (self.isDeviceAgnostic) {
+    UIDevice *device = [UIDevice currentDevice];
+    NSString *model  = [device.model stringByReplacingOccurrencesOfString:@" " withString:@"_"];
+    NSString *os     = [device.systemVersion stringByReplacingOccurrencesOfString:@"." withString:@"_"];
+
+    fileName         = [NSString stringWithFormat:@"%@_%@%@", fileName, model, os];
```

Comment:
I thought I added a test case for this but it looks like I only added it to ashfurrow/Nimble-Snapshots#32

I'll separate the formatting logic elsewhere (more so because I wanted to actually have `iPhone 5s` as the model and that would required an extension on `UIDevice`) and will correctly show the desired model and will also add a test for it tomorrow morning